ddp-workspace/projects/ddp-dsm-ui/src/app/STORE/selectors/Utils/participantsList.util.ts

```diff
@@ -11,9 +11,51 @@ export const generateParticipantsList = (participants: ParticipantModel[], setti
     birthdate: '01/01/2022',
     registered: '02/02/2002',
     lastUpdated: '01/01/2022',
-    activities: pt.esData.activities.map(activity => ({
-      activityName: Object.values(actDefs).find(actDef => actDef['activityCode'] === activity.activityCode)['activityName'],
-      activityGuid: activity.guid
-    }))
+    activities: pt.esData.activities.map(activity => {
+      const activityGot = Object.values(actDefs).find(actDef => actDef['activityCode'] === activity.activityCode);
```

Comment:
I am trying to guess what `activityGot` means.

Reply:
May be `activityDefinition` ?
It will be quite hard to understand later what was the `Got` meaning.